applications/matter_bridge/Kconfig

```diff
@@ -53,7 +53,7 @@ config BT_SCAN_FILTER_ENABLE
 
 # Configure how many Bluetooth LE service UUIDs the Matter bridge can scan.
 config BT_SCAN_UUID_CNT
-	default 1
+	default 2
```

Comment:
Why not 3 since we have temperature, humidity, and onoff uuids support? Unless the Environmental Sensor has one UUID, then it is correct.

Reply:
I think it is correct, as ES is one service containing few different characteristics.